diff --git a/Source/AO/Private/Character/AO_PlayerCharacter.cpp b/Source/AO/Private/Character/AO_PlayerCharacter.cpp
index 0577fa1..ef1e3a9 100644
--- a/Source/AO/Private/Character/AO_PlayerCharacter.cpp
+++ b/Source/AO/Private/Character/AO_PlayerCharacter.cpp
@@ -323,6 +323,7 @@ void AAO_PlayerCharacter::SetupPlayerInputComponent(UInputComponent* PlayerInput
 		EIC->BindAction(IA_Jump, ETriggerEvent::Started, this, &AAO_PlayerCharacter::StartJump);
 		EIC->BindAction(IA_Jump, ETriggerEvent::Triggered, this, &AAO_PlayerCharacter::TriggerJump);
 		EIC->BindAction(IA_Walk, ETriggerEvent::Started, this, &AAO_PlayerCharacter::HandleWalk);
+		EIC->BindAction(IA_Crouch, ETriggerEvent::Started, this, &AAO_PlayerCharacter::HandleCrouch);
 
 		if (IsValid(AbilitySystemComponent))
 		{
